website/docs/guides/sl-snowflake-qs.md

```diff
@@ -449,37 +449,9 @@ packages:
 
```

Comment:
hey @matthewshaver above this line ^ I see this dbt_utils reference:

> ```sql
> packages:
>  - package: dbt-labs/dbt_utils
>    version: 1.1.1
> ```

Is this the old way and should it be replaced?